uber/models.py

```diff
@@ -1085,17 +1157,25 @@ class Game(MagModel):
     code        = Column(UnicodeText)
     name        = Column(UnicodeText)
     attendee_id = Column(UUID, ForeignKey('attendee.id'))
-    attendee    = relationship(Attendee, backref='games', cascade='delete')
+    attendee    = relationship(Attendee, backref=backref('games', cascade='all,delete-orphan'))
     returned    = Column(Boolean, default=False)
-    checked_out = relationship('Checkout', backref='game', uselist=False, cascade='delete')
+
+    @property
+    def checked_out(self):
+        try:
+            return [c for c in self.checkouts if not c.returned][0]
+        except:
+            pass
 
     _repr_attr_names = ['name']
 
 class Checkout(MagModel):
```

Comment:
Note to self: Anthro also doesn't use this feature, though they could do a ``DROP TABLE "checkout";`` if they wanted to get the latest.